nautilus_trader/data/aggregation.pxd

```diff
@@ -66,6 +66,7 @@ cdef class BarAggregator:
     cdef object _handler_backup
     cdef bint _await_partial
     cdef bint _batch_mode
+    cdef public bint is_live
```

Comment:
Could something like `is_running` more clearly distinguish from an environment context? (not backtest vs live etc)

Reply:
ok, I've modified to is_running